### DISCUSSION TITLE
Tech Talks: What is new in Ant Media Server v2.9.0?

### DISCUSSION THREAD

**Discussion by zerostack0** [ant-media/Ant-Media-Server#6319] [Announcements]
![Designer](https://github.com/ant-media/Ant-Media-Server/assets/23143596/1f2ff27c-4d07-4b83-b162-b197148fc289)

🚀 Exciting News for Streaming Enthusiasts! 🚀

Join us as we dive into the latest and greatest features of Ant Media Server v2.9.0! Whether you're a developer, a content creator, or a streaming service provider, you won't want to miss this!

Key Details:

    📅 Date: May 16th
    ⏰ Time: 3:00 PM GMT+3
    🌐 Location: https://meet.antmedia.io/Conference/techtalks

🔗 Save the Date: Don't miss out! [Add to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NjUyOW81bDl0NHRvc21zcms4M3ZtbnN1cmkgY19kZjQ5YzExZWMyNDhhYWUxMjQ3ZTIzNTFlZjBkNzU1OGNlNDgxYzg4MDZjMWFmNjRmYjIzZTQyMzIzNjQ2ZTZkQGc&tmsrc=c_df49c11ec248aae1247e2351ef0d7558ce481c8806c1af64fb23e42323646e6d%40group.calendar.google.com)

Looking forward to seeing you there! 💻

**Comment by brixxnatt** [comment]
no one in the meeting?!!

**Reply by zerostack0** [reply]
It's started 1 hours ago and finished 30 mins ago.

**Reply by zerostack0** [reply]
You can check the presentation here: https://docs.google.com/presentation/d/1SWfdkAKiIbUj7kXK9jGbCVRuBo0kTutx0QDiDCxeruU/edit?usp=sharing

If you wanted to talk about any of these features, we can make a 1-1 meeting. @brixxnatt

**Comment by brixxnatt** [comment]
???

**Reply by yashtandon113** [reply]
@brixxnatt 
Do you have any questions other than those above? @mustafaboleken shared the presentation link with you.
